### DISCUSSION TITLE
关于 date 时间不修改倡议

### DISCUSSION THREAD

**Discussion by KimYangOfCat** [FEDarling/fe-translate-weekly#187] [项目讨论]
每篇周刊都有一个date时间，其在周刊生成时会自动生成，不建议按翻译的时间修改，而应该保持原有的文件生成时间，因为当有多篇周刊待翻译时，翻译顺序是不定的，由此翻译时间也不是线性的，这样会造成周刊网页端排序混乱。 参考[CSS 周刊错误的顺序排序](https://weekly.fedarling.com/categories/CSS%20Weekly/)
@daodaolee

**Comment by daodaolee** [comment]
逆序排序阅读起来确实会比较舒服，后续 YAML 的 `date` 不作修改。
